AppCenter/AppCenter/Internals/Context/UserId/MSUserIdContext.h

```diff
@@ -19,11 +21,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic) NSMutableArray<MSUserIdHistoryInfo *> *userIdHistory;
 
+@property(nonatomic) NSHashTable<id<MSUserIdContextDelegate>> *delegates;
+
+@property(nullable, nonatomic) NSString *mUserId;
```

Comment:
Use the style from this SDK, not from Android

Reply:
I removed this one. Resolving